.github/workflows/cpp.yml

```diff
@@ -175,10 +175,15 @@ jobs:
       ARROW_BUILD_STATIC: OFF
       ARROW_BOOST_USE_SHARED: OFF
       ARROW_VERBOSE_THIRDPARTY_BUILD: OFF
-      BOOST_SOURCE: BUNDLED
+      BOOST_ROOT: C:\local\boost_1_67_0
+      BOOST_LIBRARYDIR: C:\local\boost_1_67_0\lib64-msvc-14.1\
     steps:
       - name: Disable Crash Dialogs
         run: reg add "HKCU\SOFTWARE\Microsoft\Windows\Windows Error Reporting" /v DontShowUI /t REG_DWORD /d 1 /f
+      - name: Installed Packages
+        run: choco list -l
```

Comment:
Should we keep this step?

Reply:
I think it's useful to have system info in CI builds.